extensions/blocks/map/index.js

```diff
@@ -37,5 +39,60 @@ export const settings = {
 			migrate: attributes => ( { ...attributes, showFullscreenButton: true } ),
 			save,
 		},
+		{
+			attributes: {
+				mapStyle: {
```

Comment:
I may have missed it in the wider code, but I can't see where this attribute is getting set. As it is based on `className`, could it be derived in the `save` function too?